src/core/runtime/add-from-fs-nodejs.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const promisify = require('promisify-es6')
+const pull = require('pull-stream')
+const globSource = require('../../utils/files/glob-source')
+const isString = require('lodash/isString')
+
+module.exports = self => {
+  return promisify((...args) => {
+    const callback = args.pop()
+    const options = isString(args[args.length - 1]) ? {} : args.pop()
```

Comment:
This should be enough we don't box strings 
```suggestion
    const options = typeof args[args.length - 1] == 'string' ? {} : args.pop()
```

Trying to avoid anything from lodash lots of unnecessary use of it that increase bundle size.